src/get_gateway_ip.c

```diff
@@ -77,9 +77,7 @@ void parseRoutes(struct nlmsghdr *nlHdr, struct route_info *rtInfo,
 	struct rtmsg *rtMsg;
 	struct rtattr *rtAttr;
 	int rtLen;
-	char *tempBuf = NULL;
 
-	tempBuf = (char *)malloc(100);
```

Comment:
This one was cool. I wasn't sure why `-fsanitize=address` did complain about this. I guess it detects unused memory allocations as leeks as well.